cli/pcluster/cw-dashboard-substack.cfn.yaml

```diff
@@ -117,6 +115,9 @@ Resources:
 
               # EBS metrics graphs
               - ',{"type":"text","x":0,"y":21,"width":24,"height":1,"properties":{"markdown":"\n# EBS Metrics\n"}}'
+
+              {%- set NumberOfEBSVol = config.NumberOfEBSVol|int -%}
+              {%- set VolumeType = config.VolumeType.split(',') %}
```

Comment:
if it's an array I'd call it ebs_volume_types

Reply:
right, done